pyvocz/templates/index.html

```diff
@@ -20,13 +20,14 @@
             <div>
                 Pyva jsou setkání příznivců programovacího jazyka <a href="http://python.cz">Python</a>
                 a spřízněných technologií.
-                Pravidelně jsou pořádány v Praze, Brně a Ostravě.
+                Pravidelně jsou pořádány v Praze, Brně, Ostravě a za nedlouho
```

Comment:
https://www.pravidla.cz/pravopis/27309-zanedlouho.html ... Anebo "brzy", je to kratší... :)

Reply:
Fixed